src/err/mod.rs

```diff
@@ -182,6 +182,19 @@ impl PyErr {
         PyErr::from_state(PyErrState::lazy(ty, args))
     }
 
+    /// Deprecated form of [`PyErr::from_value_bound`].
+    //#[cfg_attr(
+    //    not(feature = "gil-refs"),
+    //    deprecated(
+    //        since = "0.21.0",
+    //        note = "`PyErr::from_value` will be replaced by `PyErr::from_value_bound` in a future PyO3 version"
+    //    )
+    //)]
```

Comment:
Deprecation needs to be uncommented :)

(Or was the intention to defer that for a future PR?)

Reply:
No - I commented it to silence the warnings temporarily and then forgot to uncomment.